### ISSUE TITLE
Any query causes server error "Class 'SearchWP\\Mod' not found"

### ISSUE THREAD

**Issue by heggemsnes** [funkhaus/ql-search#10]
**Describe the bug**
Any query causes internal server error:

`"Class 'SearchWP\\Mod' not found`

**To Reproduce**
Install required plugins and try a query like 
```query MyQuery {
  searchWP(where: {input: "test"}) {
    nodes {
      databaseId
    }
  }
}
```

**Expected behavior**
Results should show up

**Plugin versions**

SearchWP 3.1.16
WP Graphql 1.6.10
QL Search 1.0.0

**Comment by drewbaker** [comment]
Yeah I’d have to do some digging but this will have stopped working newer versions of SearchWP and WP-GQL. They both went through big breaking changes after this plugin was made. If you can downgrade your plugins.
